### PR TITLE
Clarified that 'rm' removes data and containers

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
As per issue - https://github.com/docker/compose/issues/190

Signed-off-by: Chris Ward <chris@crate.io>

### CODE REVIEW

**Comment by dnephin** [comment]
@ChrisChinchilla  sorry for the slow response. I think it would be good to clarify this. You'll need to update this PR since we've moved this into a new file `docs/reference/rm.md`

@moxiegirl any suggestions for the wording here?

File: docs/cli.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Pulls service images.
 
 ### rm
 
-Removes stopped service containers.
+Removes stopped service containers. _This removes the containers completely, you will lose any data that was contained on them_.
```
Comment (moxiegirl) [inline docs/cli.md]: Removes stopped service containers. Ay data stored in the container's local storage is lost when a container is removed.

**Comment by moxiegirl** [comment]
@ChrisChinchilla Thanks! @dnephin I don't know if we need to mention mounted data persists or not.

**Comment by ChrisChinchilla** [comment]
@dnephin @moxiegirl Do you still need me to do anything? Sorry, this was all a while ago :)

**Comment by dnephin** [comment]
You'll need to rebase with master, the file was moved

**Comment by ChrisChinchilla** [comment]
@dnephin done. I'm getting the signing error above ^ and the steps don't work for me, but reading this (https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work) it implies that the sign-off I did at the beginning here is still valid.

**Comment by aanand** [comment]
Every non-merge commit needs to be signed off. You've currently got 3 commits: one signed, one merge commit, one unsigned.

The cleanest thing would be to rebase on the current master so there's a single commit, make sure it's signed off, and force-push that to your fork.

**Comment by moxiegirl** [comment]
@ChrisChinchilla Explanation of the kind of rebase action JIC: http://docs.docker.com/opensource/workflow/work-issue/#pull-and-rebase-frequently

**Comment by aanand** [comment]
Closing in favour of #2554, now that we've got a `docker volume` command. Thanks for the PR!

**Pull request closed without merging**